.github/workflows/build-wheels.yml

```diff
@@ -7,124 +7,66 @@ on:
   pull_request:
     # Check all PR
 
-env:
-  SPHERICART_NO_LOCAL_DEPS: "1"
-  MACOSX_DEPLOYMENT_TARGET: "11.0"
-
 jobs:
   build-wheels:
-    runs-on: ${{ matrix.os }}
-    name: ${{ matrix.os }}
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        # TODO: add windows builder
-        os: [ubuntu-20.04, macos-13]
-
+        arch: ["x86_64"]
+        python-version: ["3.12"]
+        cuda-version: ["12.1"]
+    env:
+      CIBW_SKIP: cp36-* cp37-* cp38-* cp39-* cp310-* cp311-*
```

Comment:
I think there is a way to explicitly include just one `cpxy-`, which should be better than skipping all the ones we don't want